docs/website/docs/getting-started/installation.md

```diff
@@ -224,6 +225,23 @@ export PATH=$PATH:$HOME/bin
 # (Optional) Add the $HOME/bin to your shell initialization file
 echo 'export PATH=$PATH:$HOME/bin' >> ~/.bashrc
 ```
+</TabItem>
+<TabItem value="homebrew">
+
```

Comment:
Should we mention that this installs the latest V2.x version?

Reply:
Nope, as by default like the other installation links, it installs the latest *stable* which should be 2.X

Once we switch to 3.0.0 that'll change however.